### PR TITLE
Fix CSR

### PR DESCRIPTION
**Pull request by ckdarby**
CSR did not have the "signing" and I first cased each letter of CSR

### CODE REVIEW

File: docusaurus/docs/learn/introduction/_components.md [hunk 1]
```diff
@@ -15,7 +15,7 @@ The OpenZiti Controller must be configured with public key infrastructure
 (PKI). The configured PKI is used to create secure, mutually-
 authenticated TLS (mTLS) network connections between any two
 pieces of the OpenZiti Network. The OpenZiti Controller does not provide its
-own PKI but for the OpenZiti Controller to sign certificate requests (CSR)
+own PKI but for the OpenZiti Controller to sign Certificate Signing Request (CSR)
```
Comment (dovholuknf) [inline docusaurus/docs/learn/introduction/_components.md]: thanks. I think on this one it probably should stay lower-case and plurarize 'request'. I checked wikipedia and they don't propercase "certificate signing request".

Comment (dovholuknf) [reply docusaurus/docs/learn/introduction/_components.md]: or - "to sign a certificate signing request". That makes sense too

Comment (ckdarby) [reply docusaurus/docs/learn/introduction/_components.md]: :+1:

**Comment by ckdarby** [comment]
@dovholuknf All good, lower cased as suggested.

**Review by dovholuknf — APPROVED** [review]
thanks again!

**Pull request merged**